ci/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -eou pipefail
+
+echo '--- cargo clippy'
+cargo clippy --all --all-targets --all-features -- --deny warnings
+
+echo '--- cargo build'
+cargo build --tests --verbose --workspace
+
+echo '--- cargo test'
+cargo test --workspace
```

Comment:
looks like `cargo deny` is not run

Reply:
It is, as a separate job.

<img width="845" alt="Screenshot 2022-03-25 at 10 07 33" src="https://user-images.githubusercontent.com/158411/160090337-fab2f8eb-5c85-43a4-bf62-59d7a56ee21d.png">

Reply:
:+1: